SpatialGDK/Source/Private/Interop/SpatialSender.cpp

```diff
@@ -634,6 +639,43 @@ void USpatialSender::ResolveOutgoingRPCs(UObject* Object)
 	}
 }
 
+FString USpatialSender::GetOwnerWorkerAttribute(AActor* Actor)
```

Comment:
Is there a race condition where owner is still not resolved? In which case we return worker attribute that will be promptly invalidated.

Reply:
Potentially? This would be fixed in the next tick though.